src/components/DayPicker.jsx

```diff
@@ -361,6 +364,10 @@ class DayPicker extends React.PureComponent {
     if (!prevProps.isFocused && isFocused && !focusedDate) {
       this.container.focus();
     }
+    
+    if (prevProps.forceRender !== forceRender) {
```

Comment:
this means that going from true to false also triggers the logic

Reply:
Yes this prop is passed so that we could trigger the rerender of DatePicker component by calling onMonthChange. For example when a calendar is drawn somewhere in scrollable modal the days are not rendered, so when you scroll to DatePicker it is empty.